### PR TITLE
Changed getRecipesPowerState to getActiveRecipes

### PR DESCRIPTION
**Pull request by Webunity**
in order to match URL on brain which is called. Some files where auto-formatted by Visual Studio Code, sorry for that :)

### CODE REVIEW

**Comment by Webunity** [comment]
@MichaelKohler changes done, also in more files of the SDK where there was a space before the parenthesis...

File: lib/index.js [hunk 1]
```diff
@@ -49,12 +49,12 @@ module.exports.getRecipes = recipes.getAllRecipes;
...
  * @return {Promise} promise contains an array with powerKey's of the powered on recipes.
  * @example
- * neeoapi.getRecipesPowerState(brain)
+ * neeoapi.getActiveRecipes(brain)
  * .then((poweredOnKeys) => {
  *   console.log('- Power state fetched, powered on recipes:', poweredOnKeys);
  * })
  */
-module.exports.getRecipesPowerState = recipes.getRecipePowerState;
+module.exports.getActiveRecipes = recipes.getActiveRecipes;
```
Comment (MichaelKohler) [inline lib/index.js]: thanks for your work so far! last thing: as we don't want to break existing implementations right now, would you mind adding back the old one that points to the new function? Then existing implementations won't break

```
// dont break the SDK, @deprecated
module.exports.getRecipesPowerState = recipes.getActiveRecipes;
```

**Comment by Webunity** [comment]
@MichaelKohler done.

**Comment by MichaelKohler** [comment]
Me again, sorry :( 

1) I've tried to apply it to our developer version, but miserably failed. Would you mind updating this to the next branch and update the PR to target our next branch? That might make the whole thing easier and we don't need to manually pick it into our repository and it will always be part of the history :)
2) Something with the index.html changes went wrong here. Can you revert those changes by running`npm run doc`? That should produce an HTML file in dist/doc/ that only has the necessary changes in it. You can then move that file to docs/ and the difference to our next branch should only be the method rename. There is no need to do any code formatting there, as it's auto-generated.

Tested the changes, looks good!

**Comment by Webunity** [comment]
Done. see #127

**Pull request closed without merging**